data.js

```diff
@@ -0,0 +1,48 @@
+let Title;
+let Author;
+let bookList = [];
+
+function getValues() {
+  Title = document.querySelector('#title').value;
+  Author = document.querySelector('#author').value;
+  document.querySelector('#title').value = '';
+  document.querySelector('#author').value = '';
+}
+
+function addBook() {
+  document.getElementById('dynamic').innerHTML = '';
+  bookList.forEach((book, index) => {
+    document.getElementById('dynamic').innerHTML += `
+    <div class="books">
+      <p class="title">${book.title}</p>
+      <p class="authir">${book.author}</p>
+      <button class="remove" onClick = 'removeItem(${index})'>Remove</button>
+      <hr> 
+    </div>`;
+  });
+}
```

Comment:
- [ ] Kindly change this to es6 arrow function syntax following the JavaScript best practices guide.

Reply:
ES6 is not applicable until the next week